.circleci/config.yml

```diff
@@ -6,9 +6,10 @@ orbs:
 executors:
   node:
     docker:
-      - image: cimg/node:18.16-browsers
-        environment:
-          NODE_OPTIONS: "--openssl-legacy-provider"
```

Comment:
**question:** is this not needed anymore?

Reply:
Not sure, thought I'd run it and see 😆